tests/test_vega.py

```diff
@@ -263,7 +256,776 @@ def test_fill_anchor_in_string(tmp_dir):
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "acc": "0.1",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 2,
+                },
+            ],
+            "acc",
+            [{"filename": "test", "field": "acc"}],
+            ["rev", "acc", "step"],
+            {},
+            "datum.rev",
+            ["rev"],
+        ),
+        (
+            [
+                {
+                    "rev": "B",
+                    "acc": "0.05",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "acc": "0.1",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 2,
+                },
+                {
+                    "rev": "B",
+                    "acc": "0.04",
+                    "filename": "train",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "acc": "0.09",
+                    "filename": "train",
+                    "field": "acc",
+                    "step": 2,
+                },
+            ],
+            "acc",
+            [
+                {"filename": "test", "field": "acc"},
+                {"filename": "train", "field": "acc"},
+            ],
+            ["rev", "acc", "step", "filename"],
+            {
+                "field": "filename",
+                "scale": {"domain": ["test", "train"], "range": [[1, 0], [8, 8]]},
+                "legend": {
+                    "symbolFillColor": "transparent",
+                    "symbolStrokeColor": "grey",
+                },
+            },
+            "datum.rev + '::' + datum.filename",
+            ["rev", "filename"],
+        ),
+        (
+            [
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.05",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.1",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 2,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.04",
+                    "filename": "train",
+                    "field": "acc_norm",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.09",
+                    "filename": "test",
+                    "field": "acc_norm",
+                    "step": 2,
+                },
```

Comment:
I'm lost on this one. In table form, the data looks like this:

`test` file:

| step | acc | acc_norm |
|------|----|----|
|1| 0.05||
|2|0.1|0.09|

`train` file:

| step | acc | acc_norm |
|------|----|----|
|1||0.04|

Is that what's intended?